### ISSUE TITLE
Midterm Peer Review - axl4

### ISSUE THREAD

**Issue by rgangela99** [aliciayuting/ORIE4741-project#6]
This team is working on a project to predict which potential credit card clients will default. They are using a dataset containing information for 30000 credit card clients and whether each client defaulted on their bill. The data contains real-valued, categorical, and time-series features. From this data, the team would like to learn which attributes of credit card clients effect whether a customer will default.

Good things:
1) I thought the visualizations the team provided for the distributions of different features in the dataset provided a good overview of what the dataset looks like.
2) The team's description of how they are splitting their dataset into training, validation, and test sets and their justification for their splitting method was good. It helped me understand why they chose a stratified split rather than just a random split.
3) The team's description of how they will measure error was also good. It gave me a good basic idea of the error metrics even though I have not seen confusion matrices or ROC before.

Suggestions:
1) The team mentioned perceptron, XGBoost, random forest, and neural networks as the models they will try. I would suggest they also try using some of the loss functions for classification such as logistic loss and hinge loss with the proximal gradient descent method.
2) The team mentioned using the random forest algorithm since it avoids overfitting/underfitting. I would suggest they also think about how to reduce over/underfitting with the other methods they will try.
3) It would be good to see more of the team's interpretation of what the columns of the dataset are to be able to better judge whether their feature engineering plan makes sense. The kaggle dataset does not have descriptions for the columns, so it is hard to tell what some of the columns such as PAY_1-PAY_6 mean.
